src/global_mapper_ros/global_mapper_ros.cc

```diff
@@ -98,7 +129,7 @@ void GlobalMapperRos::PublishMap(const ros::TimerEvent& event) {
   for (int i = 0; i < global_mapper_ptr_->voxel_map_ptr_->num_cells; i++) {
     global_mapper_ptr_->voxel_map_ptr_->indToLoc(i, xyz);
 
-    if (global_mapper_ptr_->voxel_map_ptr_->readValue(xyz) > 0.0) {
+    if (global_mapper_ptr_->voxel_map_ptr_->readValue(xyz) > 0.6) {
```

Comment:
Magic number for now.  There is a much bigger change in the pipes to binarize the voxel map given some upper and lower bounds on occupancy.